tests/internal/py35/test_wrapping.py

```diff
@@ -6,6 +6,23 @@
 from tests.internal.py35.asyncstuff import async_func as asyncfoo
 
 
+def test_wrap_generator_yield_from():
+    channel = []
+
+    def wrapper(f, args, kwargs):
+        channel[:] = []
+        for _ in f(*args, **kwargs):
+            channel.append(_)
+            yield _
```

Comment:
Probably want to do some crazier stuff here that `yield from` was designed for, like raising exceptions and passing data back and forth